glusterd2/xlator/validation.go

```diff
@@ -52,6 +52,29 @@ func validateBitrot(v *volume.Volinfo, key string, value string) error {
 	return nil
 }
 
+func validateQuota(v *volume.Volinfo, key string, value string) error {
+
+	// Check if quota is already enabled
+	if volume.IsQuotaEnabled(v) == false {
+		err := fmt.Errorf("Quota not enabled to set this value: '%s'", key)
+		return err
+	}
+
+	switch key {
```

Comment:
Use the constants defined above.

    key = "quota." + key
    switch key{
    case VkeyQuotaDeemStatfs:

Reply:
As @prashanthpai  wants to remove the VkeyQuota.. options this is contradicting, need to resolve this confusion. and then will make changes as per it.

Reply:
done